src/lib.ts

```diff
@@ -615,82 +559,6 @@ export function fairyMultiplier(familiar: Familiar): number {
   return Math.pow(Math.sqrt(itemBonus + 55 / 4 + 3) - Math.sqrt(55) / 2, 2);
 }
 
-let monsterManuelCached: boolean | undefined = undefined;
```

Comment:
Moved to combat.ts to remove circular dependency (since lib needs to use embezzler)